src/NuGet.Clients/NuGet.PackageManagement.VisualStudio/Projects/CpsPackageReferenceProject.cs

```diff
@@ -214,47 +222,29 @@ public async override Task<IEnumerable<PackageReference>> GetInstalledPackagesAs
         /// </summary>
         public override async Task<ProjectPackages> GetInstalledAndTransitivePackagesAsync(CancellationToken token)
         {
-            var packageSpec = GetPackageSpec();
+            var reading = await GetFullRestoreGraphAsync(token);
```

Comment:
naming: reading doesn't really tell us anything about this type.

Reply:
Added type and renamed.